Jenkinsfile

```diff
@@ -24,6 +56,16 @@ node('boardfarm') {
         sh 'echo "sed -i \'$ a nameserver 8.8.4.4\' /etc/resolv.conf" > /dev/ttyUSB0'
         sh 'sleep 10'
 
+        if(params.OVERRIDE_PACKAGES) {
```

Comment:
Whats the use case where you would set PACKAGES_PATH but not OVERRIDE_PACKAGES? i.e. could OVERRIDE_PACKAGES just be dropped?